### PR TITLE
Add network configurations in hardhat

### PR DESCRIPTION
**Pull request by boyuan-chen**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Describe what your Pull Request is about in a few sentences.

Add all network configurations in hardhat and add contract addresses to `boba-addr.json`

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

- Edit hardhat file
- Add contract addresses

## Testing

Describe how to test your new feature/bug fix and if possible, a step by step guide on how to demo this.

### CODE REVIEW

**Comment by InoMurko** [comment]
is this mergable?

**Comment by boyuan-chen** [comment]
> is this mergable?

Yes!

**Review by wsdt — APPROVED** [review]

**Comment by souradeep-das** [comment]
something broke the coverage

**Comment by boyuan-chen** [comment]
> something broke the coverage

Reran the test. All good now.

File: packages/boba/register/addresses/addressBobaFuji_0xcE78de95b85212BC348452e91e0e74c17cf37c79.json [hunk 1]
```diff
@@ -34,5 +34,9 @@
   "Proxy__BobaTuringCredit": "0x4200000000000000000000000000000000000020",
   "AuthenticatedFaucet": "0x5f6D019832FA4522DB7b94A4fe0DDBb73212FAcE",
   "WBOBA9": "0xA7cad1B188Ac7E4d05644D1cf1cBcEB4aD372a9D",
-  "L2StandardTokenFactory": "0xD2ae16D8c66ac7bc1Cf3c9e5d6bfE5f76BeDb826"
+  "L2StandardTokenFactory": "0xD2ae16D8c66ac7bc1Cf3c9e5d6bfE5f76BeDb826",
+  "L1ERC1155Bridge": "0xCc982d33525EBFB14B9c504428d02082043817D7",
+  "Proxy__L1ERC1155Bridge": "0x1dF39152AC0e81aB100341cACC4dE4c372A550cb",
+  "L2ERC1155Bridge": "0x502f97eCBfc7633EA977003b4C1F5bb81Dc4458F",
```
Comment (souradeep-das) [inline packages/boba/register/addresses/addressBobaFuji_0xcE78de95b85212BC348452e91e0e74c17cf37c79.json:40]: this address does not match with the address in `boba_community/boba-node/state-dumps/bobafuji/boba-addr.json`

**Comment by InoMurko** [comment]
do you want to revisit this @Boyuan-Chen or should we close?

**Comment by boyuan-chen** [comment]
> do you want to revisit this @Boyuan-Chen or should we close?

I will close it.

**Pull request closed without merging**